build.sbt

```diff
@@ -102,10 +102,10 @@ lazy val root = Project(id = "pekko", base = file("."))
   .settings(PekkoBuild.welcomeSettings)
   .enablePlugins(CopyrightHeaderForBuild)
 
-lazy val actor = akkaModule("akka-actor")
+lazy val actor = pekkoModule("actor")
```

Comment:
not `pekko-actor`?

Reply:
So the idea is to remove the `akka-`/`pekko-` prefix for sbt project names because its redundant (not this will not effect the `jar`/artifact, which will still have the `pekko-` prefix). This is however somewhat controversial so I was planning to make a discussion on the mailing list on it (sorry I have been a bit busy lately and this fell off the radar, will try to send off an email by end of the weekned).

Reply:
take it easy, I'm busy at 12.12 shopping festival too:)